core/src/trezor/crypto/slip39.py

```diff
@@ -414,22 +590,20 @@ def _decode_mnemonics(
     group_thresholds = set()
     group_counts = set()
 
-    # { group_index : [member_threshold, set_of_member_shares] }
+    # { group_index : [threshold, set_of_member_shares] }
     groups = {}  # type: MnemonicGroups
     for mnemonic in mnemonics:
-        identifier, iteration_exponent, group_index, group_threshold, group_count, member_index, member_threshold, share_value = decode_mnemonic(
-            mnemonic
-        )
-        identifiers.add(identifier)
-        iteration_exponents.add(iteration_exponent)
-        group_thresholds.add(group_threshold)
-        group_counts.add(group_count)
-        group = groups.setdefault(group_index, (member_threshold, set()))
-        if group[0] != member_threshold:
+        mnemonic = decode_mnemonic(mnemonic)
```

Comment:
Consider calling this for example "share" instead of mnemonic. Should fix some mypy warnings.